app/views/users/_user.html.haml

```diff
@@ -16,3 +16,7 @@
     - if expire_date
       %span{ class: expire_date_css_class(expire_date) }
         = expire_date
+  %td.delete
+    = link_to "<span class='glyphicon glyphicon-trash text-danger'>".html_safe,
+              user_path(user), method: :delete, title: "delete #{user.full_name}",
+              :data => {:confirm => "Delete #{user.full_name}?"}
```

Comment:
need translation for confirmation message:  `:confirm => "#{t(delete...)} #{user.full_name}?"`